Dockerfile.rov

```diff
@@ -18,16 +18,17 @@ COPY setup/rov_entrypoint.sh .
 #     rosdep install --ignore-src -y src/rov/
 RUN . /opt/ros/$ROS_DISTRO/install/setup.sh && \
```

Comment:
It's like a breath of fresh air by comparison
Honestly though, major kudos for forging through the above